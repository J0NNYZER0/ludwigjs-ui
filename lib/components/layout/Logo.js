import React, { Component } from 'react'
import { NavLink } from 'react-router-dom'

class Logo extends Component {

  constructor(props) {
    super(props)

  }

  render() {
    return (
      <NavLink exact activeStyle={{fontWeight: 'bold'}} to='/'>
        <span className='logo' />
      </NavLink>)
  }
}

export default Logo
