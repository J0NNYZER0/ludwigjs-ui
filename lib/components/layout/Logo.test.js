import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom'
import Logo from './Logo';

it('Logo renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Logo />
      </Router>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
