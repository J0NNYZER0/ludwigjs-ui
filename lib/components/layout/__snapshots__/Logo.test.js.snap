// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<a
  aria-current="page"
  className="active"
  href="/"
  onClick={[Function]}
  style={
    Object {
      "fontWeight": "bold",
    }
  }
>
  <span
    className="logo"
  />
</a>
`;
